.vscode/settings.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
hmm....unexpected file? :-p

Reply:
No, it's for vscode configuration.

Reply:
Yeah I thought it'd be good to include it, but if it's not needed for the simple app, then I can remove it.